pareto/examples/Membrane_Distillation/MD_PARETO.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Membrane Distillation (MD) is an emerging technology capable of treating highly saline wastewater, particularly relevant for handling hypersaline produced water. MD leverages a temperature gradient across a hydrophobic membrane to induce vapor pressure differences, facilitating the separation of clean water from a saline solution. The process operates at lower temperatures compared to traditional thermal desalination methods, making it well-suited for utilization of waste heat sources.\n",
+    "\n",
+    "This flowsheet simulates a single Direct Contact Membrane Distillation (DCMD) model operating in continuous recirculation mode to achieve high recoveries. This mode helps overcome the low single-pass recovery limitation of MD technologies, which is typically a maximum of 10%.\n",
+    "\n",
+    "The simulation includes an integrated external heat exchanger for heat recovery, optimizing energy use. The flowsheet encompasses:\n",
+    "\n",
+    "An MD module for the distillation process.\n",
+    "A heater to preheat the recycle stream and makeup feed.\n",
+    "A chiller to cool the feed to the cold side of the membrane, ensuring efficient separation driving force and condensation.\n",
+    "An external heat exchanger to recover heat from the heated pure water in the cold loop and preheat the feed to the hot channel before it enters the heater.\n",
+    "Pumps, separators, and mixers are included to support the process flow and integration.\n",
+    "\n",
+    "To simulate the performance of treating produced water using Membrane Distillation (MD), this model approximates the properties of produced water with those of an NaCl solution. This approximation is for simplification and to leverage existing property packages, providing a basis for understanding the behavior and treatment efficacy of highly saline wastewater in MD systems. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To effectively run this Jupyter Notebook and utilize the associated flowsheet for the Membrane Distillation process simulation and optimization, it's necessary to have the WaterTAP package installed on your system. Detailed installation instructions can be found inthe [WaterTAP documentation](https://watertap.readthedocs.io/en/latest/getting_started.html).\n",
```

Comment:
can be found `in the` typo

Another suggestion is to link to the installation section instead of the whole page. https://watertap.readthedocs.io/en/latest/getting_started.html#install-on-linux-windows-and-mac-arm